app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -1,41 +1,57 @@
+    <string name="string_old_way_alert_dialog_title">你确定吗？</string>
+    <string name="string_new_way_alert_dialog_title">重启！</string>
     <string name="string_yes">是</string>
     <string name="string_no">否</string>
+    <string name="string_title_select_apps">选择应用</string>
+    <string name="string_title_select_apps_screen">选择要隐藏哪些应用的 %s 通知</string>
+    <string name="string_all_apps">所有应用</string>
+    <string name="string_settings_small_donation">小额捐赠</string>
+    <string name="string_settings_large_donation">大额捐赠</string>
+    <string name="string_settings_follow_me">关注我</string>
+    <string name="string_settings_notification_access_permission">通知访问权限</string>
+    <string name="string_settings_permission">权限</string>
+    <string name="string_settings_support_me_with">支持方式 ......</string>
+    <string name="string_settings_running_in_the_background">应用当前在后台运行</string>
+    <string name="string_settings_source_code">源代码</string>
+    <string name="string_settings_twitter">推特</string>
+    <string name="string_settings_hide_app_icon_from_launcher">从启动器隐藏应用程序图标</string>
+    <string name="string_settings_settings">设置</string>
+    <string name="string_settings_select_apps">点按选择应用</string>
+    <string name="string_settings_use_old_hiding_method">使用旧的隐藏方法</string>
+    <string name="string_settings_old_way_summary">如果你有一台带主动显示功能的一加设备或任何其他与隐藏通知相关的问题，请尝试这个选项。</string>
+    <string name="string_settings_hide_persistent_notification">隐藏其他持久通知？</string>
+    <string name="string_settings_hide_persistent_notification_summery">借助我的另一款应用程序“隐藏持久通知”，你可以隐藏任何持久通知。</string>
+
+    <string name="shared_pref_key_package_name_all" translatable="false">packageNamesAll</string>
+    <string name="shared_pref_key_package_name_current" translatable="false">packageNamesCurrent</string>
+    <string name="shared_pref_key_package_name_selected" translatable="false">packageNamesSelected</string>
+    <string name="shared_pref_name" translatable="false">myComPref</string>
+    <string name="shared_pref_key_granted" translatable="false">granted?</string>
+    <string name="shared_pref_key_isOldWay" translatable="false">isOldWay</string>
+    <string name="shared_pref_key_isIconHidden" translatable="false">isIconHidden</string>
+    <string name="string_all_key" translatable="false">all.all.all.all.all</string>
+
+    <string name="string_filter_intent" translatable="false">com.iboalali.sysnotifsnooze.NOTIFICATION_LISTENER_SERVICE_INTENT</string>
+    <string name="shared_pref_key_version_code" translatable="false">version_code</string>
 
     <!-- To all contributors, Please do not translate these two. I will use the translation used by AOSP.
             If there are any questions or change requests about these translations please contact me -->
-    <string name="string_app_name_replace_running_in_the_background">正在后台运行</string>
-    <string name="string_app_name_replace_using_battery">正在消耗电量</string>
```

Comment:
These two should not be translated by anyone, as mentioned in the comment.
I will use the official AOSP translation for it.

Reply:
@iboalali They are not translated.

Reply:
https://github.com/sr093906/sysnotifsnooze/blob/a08f32d5914335c85eec69338e288f8d10ec04b6/app/src/main/res/values-zh-rCN/strings.xml

Reply:
Sorry 😓 looked at the wrong changes